README.md

```diff
@@ -60,20 +54,19 @@ Most likely you didn't get that sick dev job, and are doing QA, or IT, or someth
+If you do get an offer: negotiate. They will _never_ take away your offer because you tried to negotiate. Always ask for a few dollars more, especially if you have a competing offer. Just ask nicely, "Hey, could you increase the salary from 20 to 24 an hour? I would feel a lot more confident about my decision if you could do that for me" or "I have an offer from AutoSoft, and while I would love to work with you, I couldn't justify it if I could get more there". If they say something like "HR won't let me" that's a lie, but you probably don't have enough leverage to negotiate anyway. I used to negotiate for jobs (and return offers) I knew I wasn't going to take just to practice or get better leverage. This advice is more important for fulltime, but it's good to get into the habit now.
 
 It can feel super scary to ask for more, especially if you feel lucky to even get the offer. I still get too scared, but in retrospect it's always worth it. More [info for fulltime here](https://www.kalzumeus.com/2012/01/23/salary-negotiation/).
 
 ### On the job
-Nice, you got a job! Dedicate yourself to this coop, ask as many questions as you can, and leech knowledge from your coworkers. Ask for projects you think are too hard for you, and then ask for help. Looking stupid doesn't matter if the first three months. There's this type of person that asks a lot of basic questions for months and before you know it, they know everything and you're coming to them for help. It's good to be this type of person.
+Good _job_, you got a job! Dedicate yourself to this coop, ask as many questions as you can, and leech knowledge from your coworkers. Ask for projects you think are too hard for you, and then ask for help. Looking stupid doesn't matter if the first three months. It's good to be the type of person that asks a lot of basic questions for months, and before you know it, they know everything and people are coming to them for help.
 
 But also: now is the time to start worrying about your next job. You want to make your resume real nice for your next internship, and the easiest way to do that is to start planning now. Look for projects that have _measureable_ impact and look good. Aim for projects or tasks that lend themselves to interview lines, something that'll get you a "23% faster" or "2x increase in" in your job bullet points. Move the needle on some quantifiable metrics.
 
+## Misc
+If this guide helped please let me know. Email me, hit me up on [twitter](twitter.com/zvorygin) or whatever else works for you. If it didn't help or you disagree, please reach out as well.
 
-### Misc
-If this guide helped please let me know. Either email me, hit me up on [twitter](twitter.com/zvorygin) or whatever works for you. If it didn't help or you disagree, please reach out as well.
-
-I accept PR's.
+PRs are welcome.
```

Comment:
not sure why we're changing this

Reply:
i moderately carefully controlled the tone of this document, this is a deviation from that tone